### ISSUE TITLE
As a Commercial user I would like to be able to submit basic information to request access

### ISSUE THREAD

**Issue by srconway** [UserOfficeProject/user-office-project-issue-tracker#1210]
Request from IDG Oct 24

Could there be a commercial access route in the normal system - with minimal non-sensitive questions e.g. who are you, what's your experiment title which would put the control for adding CoIs etc with the PI rather than as it currently does relying on the instrument scientists to add an 'Approved Experiment' and define who can see it.

This could have wider implications which would need to be considered:

- Embargo periods - could we still identify these as commercial and ensure the embargo period on the data is kept as 'indefinite' rather than the standard 3-year embargo on academic experiments. (Question for IDAaaS/ICAT?)

- Would this cause any more information about the experiment to be exposed anywhere - is the title itself sensitive and could we make that clear enough to the user so they can provide a suitably obscure title?

**Comment by mutambaraf** [comment]
@srconway  have we any update on this.

**Comment by srconway** [comment]
No, we'll need a steer from @Cat-Lucifer about the requirements here and whether this is something we want to pursue. Thinking about it again the embargo shouldn't be an issue if we're generating suitable proposal numbers.

**Comment by ellen-wright** [comment]
Not currently needed, however if commercial access is needed in the future no changes will be needed as long as we use commercial RB prefixes and suitably vague questions
